modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding-upgrade-viability-conditiontype_{context}"]
+= Understanding upgrade viability with conditionType
+
+The status of cluster operators is revealed with the value of conditionType, informing you of potential impacts to upgrading your cluster. 
+This information can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+[source,terminal]
+----
+$ oc get clusterversion -o YAML
+----
+
+* Available: 
+The conditionType `Available` indicates that the operators are functional and available in the cluster. If the status is `Available=False`, then the status indicates that at least one part of the operand is non-functional and the condition requires an administrator to intervene.
```

Comment:
Can we speak about "condition with type `Available`? Speaking about conditionTypes in sentences looks really awkward.

Reply:
I think it would also probably use a "If its status is `False`" instead of the "status is `Available=False`. Condition is an entity which has a type (Available) and status (True/False) and more (irrelevant here).